tools/flaky-test-reporter/result.go

```diff
@@ -68,18 +75,36 @@ func (ts *TestStat) hasEnoughRuns() bool {
 	return len(ts.Passed) + len(ts.Failed) >= requiredCount
 }
 
-func getFlakyRate(testStats map[string]TestStat) (float32, error) {
-	totalCount := len(testStats)
-	if 0 == totalCount {
-		return 0.0, nil
+func (ts *TestStat) getTestStatus() string {
+	switch {
+		case ts.isFlaky():
+			return flakyStatus
+		case ts.isPassed():
+			return passedStatus
+		case !ts.hasEnoughRuns():
+			return lackDataStatus
+		default:
+			return failedStatus
 	}
-	flakyCount := 0
-	for _, ts := range testStats {
+}
+
+func getFlakyTests(rd *RepoData) []string {
+	var flakyTests []string
+	for testName, ts := range rd.TestStats {
 		if ts.isFlaky() {
-			flakyCount++
+			flakyTests = append(flakyTests, testName)
 		}
 	}
-	return float32(flakyCount)/float32(totalCount), nil
+	return flakyTests
+}
+
+
+func getFlakyRate(rd *RepoData) (float32, error) {
```

Comment:
you are only returning nil as err. Do we need this as a return var?

Reply:
removed